sdk/src/main/scala/com/horizen/storage/ErgoPlatformStorageAdapter.scala

```diff
@@ -0,0 +1,72 @@
+package com.horizen.storage
+import java.io.File
+import java.util
+import java.util.{Optional, List => JList}
+
+import com.horizen.utils._
+import javafx.util.{Pair => JPair}
+import org.ergoplatform.db.LDBFactory.factory
+import org.ergoplatform.db.VersionedLDBKVStore
+import org.iq80.leveldb.Options
+
+import scala.collection.JavaConverters._
+import scala.compat.java8.OptionConverters._
+
+
+/*
+*  @TODO to discuss
+*    1. Why we use ByteArrayWrapper instead of Array[Byte]?
+*    2. We need iterator over the storage
+* */
+class ErgoPlatformStorageAdapter(pathToDB: String, keepVersions: Int) extends Storage{
```

Comment:
Why not to replace "ErgoPlatform..." to "VersionedLevelDB..." or something like that